code/modules/keybindings/bindings_client.dm

```diff
@@ -88,7 +88,7 @@
 	// can hold different keys and releasing any should be handled by the key binding specifically
 	for (var/kb_name in prefs.key_bindings[_key])
 		var/datum/keybinding/kb = GLOB.keybindings_by_name[kb_name]
-		if(kb.up(src))
+		if(kb.can_use(src) && kb.up(src))
```

Comment:
Was checked for keydown, but not keyup

https://github.com/tgstation/tgstation/blob/add74bee56cfde91602eb8ee28e3cae5959bc4f6/code/modules/keybindings/bindings_client.dm#L65-L72